### ISSUE TITLE
Screen doesn't reflect sort order imediately after reload

### ISSUE THREAD

**Issue by rpolve** [newsboat/newsboat#1089]
**Version:**

newsboat 2.20.1

**Config file:**

```
auto-reload yes
feed-sort-order lastupdated
```

**Steps to reproduce:**

1. run newsboat

2. wait for auto-reload to complete

3. open first entry

**Expected behaviour:**

Open the highlighted feed

**Actual behaviour:**

Open the feed which after reload is the newest, as if the feed were sorted but the screen didn't reflect this.

**Other info**:

Going back to the list of feeds shows the correct sort order at this point.

```
newsboat 2.20.1
System: Linux 5.7.7-arch1-1 (x86_64)
Compiler: g++ 10.1.0
ncurses: ncurses 6.2.20200212 (compiled with 6.2)
libcurl: libcurl/7.71.1 OpenSSL/1.1.1g zlib/1.2.11 libidn2/2.3.0 libpsl/0.21.0 (+libidn2/2.2.0) libssh2/1.9.0 nghttp2/1.41.0 (compiled with 7.70.0)
SQLite: 3.32.3 (compiled with 3.32.3)
libxml2: compiled with 2.9.10
```

**Comment by Minoru** [comment]
Reproduced with https://github.com/newsboat/newsboat/commit/47137b97d09d1398c6950c8d73745966d9ed0845 (current HEAD). Thanks for the report! I'll look into this.

**Comment by Minoru** [comment]
@rpolve, if you can build from Git, please try my PR #1090. Otherwise the fix should be available in Newsboat 2.21, less than three months from now.

**Comment by rpolve** [comment]
Thank you @Minoru, I can confirm it now works as expected.
